lib/Controller/MyOCSController.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+namespace OCA\Tables\Controller;
+
+use Exception;
+use OCA\Tables\AppInfo\Application;
+use OCA\Tables\Errors\InternalError;
+use OCA\Tables\Errors\NotFoundError;
+use OCA\Tables\Errors\PermissionError;
+use OCP\AppFramework\Http;
+use OCP\AppFramework\Http\DataResponse;
+use OCP\AppFramework\OCSController;
+use OCP\IRequest;
+use Psr\Log\LoggerInterface;
+
+class MyOCSController extends OCSController {
```

Comment:
```suggestion
abstract class AOCSController extends OCSController {
```

To make it clear that this is a base class that will not be initialized on its own? The `A*` is a pattern we use in some other code parts in server to indicate abstract classes in the naming. I'd prefer that more over the `My` which doesn't have much meaning from my perspective.